ush/config_defaults.sh

```diff
@@ -266,6 +267,7 @@ OBSPATH="/public/data/grids/rap/obs"
 OBSPATH_NSSLMOSIAC="/public/data/radar/mrms"
 LIGHTNING_ROOT="/public/data/lightning"
 ENKF_FCST="/lfs4/BMC/public/data/grids/enkf/atm"
+FFG_DIR="/public/data/grids/ncep/ffg/grib2"
```

Comment:
@EricJames-NOAA Is this FFG product publicly available?  Also, we may want to set the default value of FFG_DIR to an empty string so that the UPP can continue to run through if there is no FFG data?

Reply:
@guoqing-noaa it is publicly available.  On Jet there is a "latest.FFG" file which points to the latest GRIB2 file.  Should we put a check that this FFG file exists before trying the regridding?  That way I think UPP should continue to run when the file is missing.

Reply:
@EricJames-NOAA Thanks for the information. I am working on some minor changes on exregional_run_post.sh and can add the existence check. Will request your review for the coming PR.

Reply:
@guoqing-noaa perfect!   Thank you!  I will watch for it.

Reply:
@EricJames-NOAA  Sorry that I cannot request reviewers at the PR page.  I just created a PR at https://github.com/NOAA-GSL/regional_workflow/pull/142. Welcome any feedbacks. Thanks! 
(I accidentally delete your last comment. Sorry about that)